scripts/alert_emails/send_emails.ts

```diff
@@ -78,7 +106,7 @@ async function setLastSnapshotNumber(
     return db
       .collection(`snapshots/${currentSnapshot}/locations/${fips}/emails/`)
       .doc(email)
-      .set({ sentAt: admin.firestore.FieldValue.serverTimestamp() });
+      .update({ sentAt: admin.firestore.FieldValue.serverTimestamp() });
```

Comment:
Using `set()` would clobber the `deleteAt` field we wrote previously, so we use `update()` instead.